packages/utils/src/__tests__/checkValueSatisfiesCondition.test.js

```diff
@@ -0,0 +1,128 @@
+import { checkValueSatisfiesCondition } from '../checkValueSatisfiesCondition';
+
+describe('checkAgainstConditions', () => {
+  describe('checkValueSatisfiesCondition()', () => {
+    it('should throw an error if given an unknown operator', () => {
+      expect(() => checkValueSatisfiesCondition('hello', { operator: 'NOT_VALID' })).toThrowError(
+        "Unknown operator: 'NOT_VALID'",
+      );
+    });
+
+    it.each([
+      ['hello', 'hello', true],
+      [0, 0, true],
+      [true, true, true],
+      [false, false, true],
+      [null, null, true],
```

Comment:
Nice conversion to parameterized tests in this file!!

I think it's a good call to not assign to `testData` here, but rather provide them as a direct input in `it.each` 💯